actiontext/lib/action_text/content.rb

```diff
@@ -130,6 +133,13 @@ def to_partial_path
       "action_text/contents/content"
     end
 
+    # Safely transform Content into an HTML String
```

Comment:
I think we should use the same verb form as used in the other methods:
```suggestion
    # Safely transforms Content into an HTML String.
```